decidim-core/app/forms/decidim/amendable/create_form.rb

```diff
@@ -4,50 +4,34 @@ module Decidim
   module Amendable
     # A form object to be used when users want to amend an amendable resource.
     class CreateForm < Decidim::Amendable::Form
-      mimic :amend
+      include Decidim::ApplicationHelper
+
+      mimic :amendment
 
       attribute :amendable_gid, String
       attribute :user_group_id, Integer
-      attribute :emendation_fields, Object
-      attribute :title, String
-      attribute :body, String
+      attribute :emendation_params, Object
 
       validates :amendable_gid, presence: true
-      validates :title, :body, presence: true, etiquette: true
-      validates :title, length: { maximum: 150 }
+      validates :emendation_params, presence: true
+      validate :emendation_changes_amendable
+      validate :check_amendable_form_validations
 
-      def title
-        @title ||= emendation_fields[:title]
-      end
+      def emendation_changes_amendable
+        return unless amendable.amendable_fields == [:title, :body]
+        return unless present(amendable).title == present(emendation).title
+        return unless present(amendable).body.delete("\r") == present(emendation).body.delete("\r")
 
-      def body
-        @body ||= emendation_fields[:body]
+        errors.add(:title, "AND body cannot be identical")
```

Comment:
These strings should be handled via the i18n system, have you checked this can be translated?

Reply:
I have no experience with this. Can you point me to some file where I can see how its done?